pkg/groups/interfaces.go

```diff
@@ -35,4 +37,5 @@ type OpenFGAClientInterface interface {
 	WriteTuples(context.Context, ...ofga.Tuple) error
 	DeleteTuples(context.Context, ...ofga.Tuple) error
 	Check(context.Context, string, string, string, ...ofga.Tuple) (bool, error)
+	BatchCheck(context.Context, ...ofga.Tuple) (bool, error)
```

Comment:
if we moved this in the middleware we could get rid of https://github.com/canonical/identity-platform-admin-ui/blob/main/internal/authorization/middlewares.go#L102,L118